### PR TITLE
feat: Implement profile chunk storage

### PR DESCRIPTION
**Pull request by phacops**
This will store profile chunks appropriately.

https://github.com/getsentry/snuba/pull/5895

### CODE REVIEW

File: internal/chunk/chunk.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		c.ID,
+	)
+}
+
+func (c *Chunk) StartEndTimestamps() (float64, float64) {
+	count := len(c.Profile.Samples)
+	if count == 0 {
+		return 0, 0
+	}
+	return c.Profile.Samples[0].Timestamp, c.Profile.Samples[count-1].Timestamp
```
Comment (Zylphrex) [inline internal/chunk/chunk.go:65]: Here we're assuming the samples are in chronological order, which is desired but is that guaranteed?

File: cmd/vroom/chunk.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	hub.Scope().SetContext("Profile metadata", map[string]interface{}{
+		"chunk_id":        c.ID,
+		"organization_id": strconv.FormatUint(c.OrganizationID, 10),
+		"profiler_id":     c.ProfilerID,
+		"project_id":      strconv.FormatUint(c.ProjectID, 10),
+		"size":            len(body),
```
Comment (Zylphrex) [inline cmd/vroom/chunk.go:50]: Just a thought but can we emit this as a metric to see the average payload size by platform?

**Review by Zylphrex — APPROVED** [review]

File: internal/chunk/chunk.go (hunk 1 above)
Comment (phacops) [reply internal/chunk/chunk.go:65]: https://github.com/getsentry/relay/pull/3588

File: cmd/vroom/chunk.go (hunk 2 above)
Comment (phacops) [reply cmd/vroom/chunk.go:50]: We'll wait until @viglia is done with implementing metrics then. Maybe you can even beta test on this.

**Pull request merged**